client/my-sites/customer-home/cards/actions/quick-links/index.jsx

```diff
@@ -1,4 +1,5 @@
 import i18nCalypso, { getLocaleSlug, useTranslate } from 'i18n-calypso';
+import { debounce } from 'lodash';
```

Comment:
Looks like the `use-debounce` package (which is already available in Calypso) has a `useDebounceCallback()` hook which supports the sort of usage we want after all: https://www.npmjs.com/package/use-debounce#debounced-callbacks

We should switch to `use-debounce` because
1) it means we're not adding a dependency on `lodash` which we're migrating away from
2) it handles the case where the component unmounts before the timeout is up

Reply:
We probably want to call the "Flush method" when the component unmounts.

Reply:
Sounds good 👍